index.php

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>title</title>
+</head>
+<body>
+<?php
+//This connects to my database
```

Comment:
Comments are fine for taking notes but I would remove them before merging into master